scripts/cnode-helper-scripts/env

```diff
@@ -144,6 +144,12 @@ fi
 if ! jq -r . "${CONFIG}" >/dev/null 2>&1; then
   echo "Could not parse ${CONFIG} file in JSON format, please double-check the syntax of your config, or simply download it from guild-operators repository!"
   return 1
+elif [[ "$(jq -r .TraceChainDb ${CONFIG})" != "true" ]]; then
+  echo "The ${CONFIG} file suggests that you have set TraceChainDb parameter to false. Please change it to true, as it is required to extract statistics from node."
+  return 1
+elif [[ "$(jq .options.mapBackends.cardano.node.BlockFetchDecision.peers /opt/cardano/cnode/files/config.json 2>&1)" != "null" ]]; then
```

Comment:
This will return null for all configs. `cardano.node.BlockFetchDecision.peers` have to be quoted. The jq command can also utilize the -e flag available to check exit code for if a specific query return ok result or not. Config path is also hardcoded.

So something like this could replace the if check:
```
elif jq -e .options.mapBackends.\"cardano.node.BlockFetchDecision.peers\" "${CONFIG}" &>/dev/null; then
```

Reply:
Agreed, thanks